src/components/PopularItemCarousel/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
+import CarouselItem from './Item'
+
+const PopularItemCarousel = () => {
+  return (
+    <CarouselProvider
+      visibleSlides={3}
+      naturalSlideWidth={400}
+      naturalSlideHeight={500}
+      totalSlides={5}
+      infinite={true}
+    >
+      <Slider>
+        <Slide index={0}>
+          <CarouselItem caption={"1st slide"} />
+        </Slide>
+        <Slide index={1}>
+          <CarouselItem caption={"2nd slide"} />
+        </Slide>
+        <Slide index={2}>
+          <CarouselItem caption={"3rd slide"} />
+        </Slide>
+        <Slide index={4}>
+          <CarouselItem caption={"4th slide"} />
+        </Slide>
+        <Slide index={5}>
+          <CarouselItem caption={"5th slide"} />
+        </Slide>
+      </Slider>
+      <ButtonBack className="buttonBack">{'<'}</ButtonBack>
+      <ButtonNext className="buttonNext">{'>'}</ButtonNext>
+      <DotGroup className="dotGroup" />
```

Comment:
Since you are making a reusable component, can you make this `DotGroup` appear or hide according to the `props` passing from parent?
PS: this is the carousel from `AboutUs` screen where there is no `DotGroup`.
<img width="466" alt="Screen Shot 2564-02-20 at 23 20 37" src="https://user-images.githubusercontent.com/32285706/108602007-4e5d5d00-73d2-11eb-917c-af227114c8fa.png">